ports/rp2/mpconfigport.h

```diff
@@ -275,7 +275,9 @@ extern const struct _mp_obj_type_t mod_network_nic_type_wiznet5k;
 #define MICROPY_HW_BOOTSEL_DELAY_US 8
 #endif
 
+#if PICO_RP2040 || PICO_ARM
```

Comment:
Can this just be `PICO_ARM`?

Reply:
Ironically my first version of this PR only used `PICO_ARM`, but I wasn't sure whether that would cover the Pico1, hence the dual condition.

However, I see that it's set in `lib/pico-sdk/src/rp2040.cmake` too - I'll update the PR!

Reply:
Done, thanks for the review!

Reply:
Thanks for updating!